### PR TITLE
"Valor" field accepts negative values

### PR DESCRIPTION
**Pull request by wellitonSouza**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

dojot/dojot#1674

* **Other information**:

### CODE REVIEW

**Review by thiagodpf — DISMISSED** [review]
It doesn't seem like the best way to treat negative numerical values, but if it worked, it's ok for me.

File: src/js/views/dashboard/widget/wizard/Steps/Filters/index.jsx [hunk 1]
```diff
@@ -126,7 +126,7 @@ const Index = props => {
 
   const handleChangeDynamicValue = useCallback(event => {
     const { value } = event.target;
-    setData(state => ({ ...state, dynamicValue: value }));
+    setData(state => ({ ...state, dynamicValue: value.replace(/\D/gim, '') }));
```
Comment (luisfelipefurlan) [inline src/js/views/dashboard/widget/wizard/Steps/Filters/index.jsx:129]: image

Also add negative number validation to the input.
This prevents the alternating behavior between 0 and 1 from occurring.

**Review by luisfelipefurlan — CHANGES_REQUESTED** [review]

File: src/js/views/dashboard/widget/wizard/Steps/Filters/index.jsx (hunk 1 above)
Comment (wellitonSouza) [reply src/js/views/dashboard/widget/wizard/Steps/Filters/index.jsx:129]: Done

**Review by thiagodpf — APPROVED** [review]

File: src/js/views/dashboard/widget/wizard/Steps/Filters/index.jsx (hunk 1 above)
Comment (luisfelipefurlan) [reply src/js/views/dashboard/widget/wizard/Steps/Filters/index.jsx:129]: Please also place the validation in the field id = 'lastRegs'

**Review by luisfelipefurlan — CHANGES_REQUESTED** [review]

**Review by luisfelipefurlan — APPROVED** [review]

**Review by cfrancisco — APPROVED** [review]

**Pull request merged**